sopel/db.py

```diff
@@ -87,8 +88,8 @@ class SopelDB(object):
     database. It simplifies those common operations, and allows direct access
     to the database, wherever the user has configured it to be.
 
-    When configured with a relative filename, it is assumed to be in the same
-    directory as the config."""
+    When configured with a relative filename, it is assumed to be in the directory
+    set (or defaulted to) in the core setting 'homedir'."""
```

Comment:
While we are changing docstring, can we put these `"""` on their own line? They like to be alone on their own.

Reply:
:+1: